packages/client/hmi-client/src/services/dataset.ts

```diff
@@ -56,8 +56,12 @@ async function getBulkDatasets(datasetIDs: string[]) {
  * Get the raw (CSV) file content for a given dataset
  * @return Array<string>|null - the dataset raw content, or null if none returned by API
  */
-async function downloadRawFile(datasetId: string, filename: string): Promise<CsvAsset | null> {
-	const URL = `/datasets/${datasetId}/downloadCSV?filename=${filename}`;
+async function downloadRawFile(
+	datasetId: string,
+	filename: string,
+	limit: number = 100
+): Promise<CsvAsset | null> {
+	const URL = `/datasets/${datasetId}/downloadCSV?filename=${filename}&limit=${limit}`;
```

Comment:
This is what I was talking about with the `result.csv`. You can create a new endpoints on `DatasetResource.java` to do what you need:  no limits, and only filename = 'result.csv'. instead of adding capability to `downloadRawFile()`

Reply:
Is it worth it to create a new endpoint for this in `DatasetResource.java` right now? Just asking because from our conversation with Daniel the other day, it sounds like this might just be a temporary solution.

On a separate note, the `filename` here refers to the name of the dataset that's being used as the input dataset for the calibrate. It has nothing to do with `result.csv` which is the filename for the `/simulations/{id}/result` endpoint in `SimulationResource.java`